### DISCUSSION TITLE
Unable to get Login verification mail | @duck.com

### DISCUSSION THREAD

**Discussion by naeem-qv** [vercel/community#3932] [Account Issue] [answered]
### Page to Investigate

https://vercel.com/login/email?next=%2Fdashboard%2Fstores%2Fpostgres%2Fstore_Vb5x3esXa8Ldg51p%2Fdata

### Steps to Reproduce

I'm attempting to log in to my Duck.com email account, but I've noticed that the verification email is taking significantly longer to arrive compared to my other Gmail and Outlook accounts, where it arrives almost immediately.

**Comment by amyegan** [comment, marked as answer]
Hey @naeem-qv. Did you get the verification email yet? It may have gone into a spam or junk folder after filtering, so be sure to check there as well.

**Reply by naeem-qv** [reply]
Yep! I got the email after 5-10 mins. And it wasn't in junk/spam. My concern is why is it only happening with @duck.com and not other emails such as @gmail.com, ...

**Reply by amyegan** [reply]
Glad to hear the email did eventually arrive! 

I suspect the delay is because of the way [DuckDuckGo Email Protection](https://duckduckgo.com/duckduckgo-help-pages/email-protection/what-is-duckduckgo-email-protection/) works as an email forwarding service. DuckDuckGo email protection receives emails, does some filtering, and then forwards the email to your regular inbox.  The other email providers receive messages more directly. So it makes sense that it would take a bit more time for `@duck.com` emails to reach your inbox.
